functions/src/services/contentful.ts

```diff
@@ -0,0 +1,5 @@
+import * as contentful from 'contentful-management';
+
+export const managementClient = contentful.createClient({
+  accessToken: `${process.env.CONTENTFUL_CONTENT_MANAGEMENT_TOKEN}` || '',
```

Comment:
What token is this? The private or the public one?

Reply:
This is the private one for firebase functions but I moved it now to the firebase remote config